plugins/user/membyuser

```diff
@@ -2,9 +2,18 @@
 #
 # Plugin to monitor Memory usage inspired by cpubyuser
 #
-# 2012-05-23 Sebastien Campion
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#	Add this to your /etc/munin/plugin-conf.d/munin-node:
+#		[membyuser]
+#		user root # required if /proc can't be read from by any user!
 
-LU=`ps auh | cut -d' ' -f 1  | sort -u`
+## 2012-05-23 Sebastien Campion
+# changed on 2019-08-30 by pcy <pcy@ulyssis.org>:
+# - change category from 'memory' to 'system' (so it appears next to cpubyuser)
+# - use rss instead of vsz
+# - more robust username enumeration
+
+LU=`ps ax --format uname | tail +2 | sort -u | grep -v -e "^root$"`
```

Comment:
Please use `$(...)` instead of backticks. Feel free to merge this line with the following one.

Reply:
Fixed.